src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Unix.cs

```diff
@@ -160,6 +161,10 @@ protected override void Dispose(bool disposing)
 
         protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
+            if (DiagnosticsHandler.IsEnabled())
```

Comment:
With Activity support, this method will be twice less-efficient.
```C#
s_diagnosticListener.IsEnabled(RequestWriteName) || s_diagnosticListener.IsEnabled(ActivityWriteName); 
```

So having no-argument IsEnabled() seems like a good idea